app/templates/form.hbs

```diff
@@ -1,78 +1,79 @@
-<section>
-  <div class="d-flex align-items-center flex-column justify-content-center h-100 shadow p-3 mb-5 bg-white rounded">
-    <div>
-      {{#link-to 'index'}}<img class="pull-left img-thumbnail " src="images/firm_initials.png">{{/link-to}}
-      <span class="display-5 font-weight-bold text-muted " >TAX DUE DATE SEARCH</span>
-    </div>
-      <br />
+
+<div class="d-flex centerdiv-form-wrapper logo-frame-form  flex-column rounded">
+  <div class="container ">
+      {{#link-to "index"}}<span class="text-muted   text-left">Back To Search</span>{{/link-to}}
   </div>
-</section>
+</div>
 
 <div class="container custom-locale-form align-items-right">
   <h4 class="font-weight-bold text-muted">
-    Connecticut (CT) - Form 20C
+    {{model.jurisdiction}}-{{model.name}}
   </h4>
   <br />
   <form >
     <div class="form-group row">
       <label for="" class="col-sm-2 form-control-label font-weight-bold ">Jurisdiction:</label>
       <div class="col-sm-4">
-        <span class="font-italic">Connecticut (CT)</span>
+        <span class="ffont-italic text-info text-info">{{model.jurisdiction}}</span>
```

Comment:
ffont-italic is a mystype? not sure double check please.... mmm reading some more i guess it's right?

Reply:
corrected